go-controller/cmd/ovnkube-identity/ovnkubeidentity.go

```diff
@@ -436,9 +436,9 @@ func runCSRApproverManager(ctx context.Context, leaderID string, restCfg *rest.C
 		LeaderElectionNamespace:       cliCfg.leaseNamespace,
 		LeaderElectionID:              leaderID,
 		LeaderElection:                true,
-		LeaseDuration:                 utilpointer.Duration(time.Minute),
-		RenewDeadline:                 utilpointer.Duration(time.Second * 30),
-		RetryPeriod:                   utilpointer.Duration(time.Second * 20),
+		LeaseDuration:                 utilpointer.To[time.Duration](time.Minute),
```

Comment:
This will be handled in the [k8s-1.29 rebase PR](https://github.com/ovn-org/ovn-kubernetes/pull/4108), not here. 

I'll need to rebase once that goes in.
... I'll consider this if I see that PR is taking too much to be merged.

Reply:
This should now be addressed by #4207, which I have rebased . 

Resolving.